config/nrfconnect/chip-module/Kconfig

```diff
@@ -251,3 +251,12 @@ config CHIP_LOG_SIZE_OPTIMIZATION
 	  information that is too detailed to be used in most cases. You can find
 	  full configuration enabled by this option in the
 	  platform/nrfconnect/CHIPPlatformConfig.h file.
+
+config CHIP_IPV4
+	bool "Enable IPv4 support for Matter"
```

Comment:
super nit, but FYI using "Enable sth", "Support sth" prefixes in Kconfig prompts is discouraged by Kconfig style guidelines: https://docs.zephyrproject.org/latest/build/kconfig/tips.html#id22. The reason is that it's simply redundant given that the type of the variable is bool so it's clear that enables sth.

Reply:
Got it, removed "Enable".